src/Glpi/Inventory/Request.php

```diff
@@ -251,6 +253,24 @@ public function networkInventory(mixed $data): void
         $this->network($data);
     }
 
+
+    public function getConfiguration(mixed $content)
+    {
+        $hook_response = Plugin::doHookFunction(
+            Hooks::INVENTORY_GET_CONFIGURATION,
+            ['content' => $content]
+        );
+
+        if (isset($hook_response['response']) && count($hook_response['response'])) {
+            $this->addToResponse($hook_response);
+        } elseif (isset($hook_response['errors']) && count($hook_response['errors'])) {
+            $this->addError($hook_response['errors'], 400);
+        } else {
+            //nothing expected happens; this is an error
+            $this->addError("Query '" . $this->query . "' is not supported.", 400);
```

Comment:
It's not a 400 stricto sensu, the request is correct, but not handled server side (that's why the original error gives a 501).